code/datums/uplink_item.dm

```diff
@@ -1061,6 +1061,13 @@ var/list/uplink_items = list()
 	cost = 2
 	surplus = 30
 
+/datum/uplink_item/implants/mindslave
+	name = "Mindslave Implant"
+	desc = "An implant injected into another body, forcing the vitcim to obey any command by the user for a limited time."
```

Comment:
I kinda wrote it like that so that traitors do not know when the implant runs out. We can specify the time in the changelog maybe.